scripts/fastboot-server.js

```diff
@@ -10,17 +10,25 @@ const fastbootHost = process.env.FASTBOOT_HOST || '0.0.0.0';
 const fastbootPort = process.env.FASTBOOT_PORT || process.env.PORT || '4000';
 const fastbootWorkers = process.env.FASTBOOT_WORKERS || '1';
 
-let fastbootServer = new FastBootAppServer({
+const serverOptions = {
   distPath        : fastbootDistPath,
   gzip            : enableGzip === 'true',
   host            : fastbootHost,
   port            : parseInt(fastbootPort),
   chunkedResponse : enableChunkedResponse === 'true',
   workerCount     : parseInt(fastbootWorkers)
-});
+};
+
+if (process.env.SENTRY_DSN) {
+  Sentry = require('@sentry/node');
+  Sentry.init({ dsn: process.env.SENTRY_DSN });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Sentry' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011987)